README.md

```diff
@@ -65,4 +65,14 @@ In case you have macOS M1:
 ```shell
 pip uninstall cryptography cffi
 LDFLAGS=-L$(brew --prefix libffi)/lib CFLAGS=-I$(brew --prefix libffi)/include pip install cffi cryptography rust --no-binary :all:
+```
+
+Installing & enabling pre-commit
+---------------------
+
+To automatically run checks before you commit your changes you should install and run **pre-commit**:
+```shell
+pip install pre-commit
```

Comment:
I think it should be part of Pipenv development, so not a necessary step. It will be part of the development environment.
Is there any reason that it couldn't?

Reply:
No, you are right, good point. I am adding it

Reply:
@kairoaraujo when trying to update the requirement files after installing pre-commit with pipenv we get the following error when running ```tox -re py37```:
ERROR: Cannot install -r /home/user/dev/ownca/dev-requirements.txt (line 21), -r /home/user/dev/ownca/dev-requirements.txt (line 29) and importlib-metadata==4.12.0 because these package versions have conflicting dependencies.
```
The conflict is caused by:
    The user requested importlib-metadata==4.12.0
    click 8.1.3 depends on importlib-metadata; python_version < "3.8"
    flake8 5.0.4 depends on importlib-metadata<4.3 and >=1.1.0; python_version < "3.8"
```
It seems like a common issue https://github.com/pypa/pipenv/issues/4476. 
Two workarounds I see and have tested are:
1) pinning to an older version of flake8 like 3.9.2 (not optimal)
2) stop testing/supporting in python3.7 environments (btw EOL is in Jun 2023)

Reply:
I think we still need to keep python3.7 as the EOL a bit far.
IMO, pining the version only for python3.7 is the best way to go.